docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -80,13 +80,14 @@ class [[eosio::contract]] smrtcontract : public contract {
 };
 ```
 
-### Define a unique index on property personal_id using the eosio::kv::table::index template class
+### Define a unique index on data member personal_id using the eosio::kv::table::index template class
 
 1. Use the `eosio::kv::table::index` template class with two parameters.
-2. Pass the name of the index as the first parameter. The parameter must be a qualified `eosio:name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
-3. Pass the reference to the property for which the index is defined, `&person::personal_id`, as the second parameter.
+2. The first parameter is the name of the index and it must be a qualified `eosio:name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
+3. The second parameter is the reference to the data member for which the index is defined, `&person::personal_id`.
+4. Call the base class method `init()` in the constructor of `address_table` type and pass the contract name as the first parameter and the `personal_id_idx` index as the second parameter.
```

Comment:
No macro this time?

Reply:
nope. two examples are with macros and two without.

Reply:
comment about nesting 2 and 3 under 1 as above 

comment same for step 4 as above